python/eos/analysis.py

```diff
@@ -564,6 +564,60 @@ def sample_pmc(self, log_proposal, step_N=1000, steps=10, final_N=5000, rng=np.r
 
         return samples, weights, sampler.proposal
 
+
+    def log_likelihood(self, x, *args):
+        """
+        Adapter for use with external sampling software (e.g. dynesty) to aid when sampling from the log(likelihood).
+        
+        :param x: Parameter point, with the elements in the same order as in eos.Analysis.varied_parameters, rescaled so that every element is in the interval [-1, +1].
+        :type x: iterable
+        :param args: Dummy parameter (ignored)
+        :type args: optional
+        """
+        for p, v in zip(self.varied_parameters, self._x_to_par(x)):
+            p.set(v)
+
+        try:
+            return(self._log_likelihood.evaluate())
+        except RuntimeError as e:
+            eos.error('encountered run time error ({e}) when evaluating log(posterior) in parameter point:'.format(e=e))
+            for p in self.varied_parameters:
+                eos.error(' - {n}: {v}'.format(n=p.name(), v=p.evaluate()))
+            return(-np.inf)
+
+    
+    def _prior_transform(self, u):
+        """
+        Adapter for use with external sampling software to aid when sampling from the log(prior).
+        
+        :param u: The input probability point on the hypercube [0, 1)**D
+        :type u: iterable
+        """
+        return self._par_to_x([prior.inverse_cdf(p) for prior, p in zip(self._log_posterior.log_priors(), u)])
+
+    
+    def sample_nested(self, bound='multi', nlive=250, dlogz=0.05):
```

Comment:
Is there a python way to keep dynesty's default behaviour for `bound`? Here we fix it to `multi` by default, but if I understand correctly this can be suboptimal in a couple of cases

Reply:
What exactly do you mean by "a python way"? According to the documentation, dynesty's default is also 'multi'.

Reply:
Hmm I can't find the part of the documentation I had in mind. Forget about this comment, sorry.